### PR TITLE
WIP: Extracting common quickstart texts into components

### PR DESCRIPTION
**Pull request by vcampitelli**
Extracting common texts from our quickstarts into components:

| Component | Content |
|-----------|:--------|
| `<Intro>` | _"In this quickstart, you are going to... The Docker Compose file and source code are available at [url]"_ |

### CODE REVIEW

**Comment by mooreds** [comment]
So excited about this.

**Comment by lyleschemmerling** [comment]
Two things:
 - you'll want to add `codeRoot` to astro/src/config.js as an optional frontmatter attribute to the collection schema
 - there's both `coderoot` and `codeRoot` in there. I think the camel case is preferred

File: astro/src/components/quickstarts/DockerSpinup.astro [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      )
+    }
+    <li>Your admin username is <code><RemoteValue url={kickstartUri} selector="$.variables.adminEmail" /></code> and the password is <code><RemoteValue url={kickstartUri} selector="$.variables.adminPassword" /></code>.</li>
+    <li>The base URL of FusionAuth is <code>http://localhost:9011/</code>.</li>
+</ul>
+
+<p>You can log in to the <a href="http://localhost:9011/admin">FusionAuth admin UI</a> and look around if you want to, but with Docker and Kickstart, everything will already be configured correctly.</p>
+
+<Aside type="note">
+    <p>If you want to see where the FusionAuth values came from, they can be found in the <a href="http://localhost:9011/admin">FusionAuth app</a>. The tenant ID is found on the Tenants page. To see the Client Id and Client Secret, go to the Applications page and click the <code>View</code> icon under the actions for the ChangeBank application. You'll find the Client Id and Client Secret values in the <code>OAuth configuration</code> section.</p>
```
Comment (mooreds) [inline astro/src/components/quickstarts/DockerSpinup.astro:54]:
```suggestion
    <p>If you want to see where the FusionAuth values came from, they can be found in the <a href="http://localhost:9011/admin">FusionAuth app</a>. The tenant Id is found on the Tenants page. To see the Client Id and Client Secret, go to the Applications page and click the <code>View</code> icon under the actions for the ChangeBank application. You'll find the Client Id and Client Secret values in the <code>OAuth configuration</code> section.</p>
```

File: astro/src/components/quickstarts/DockerSpinup.astro [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+</Aside>
+
+<Aside type="caution">
+    <p>The <code>.env</code> and <code>kickstart.json</code> files contain passwords. In a real application, always add these files to your <code>.gitignore</code> file and never commit secrets to version control.</p>
```
Comment (mooreds) [inline astro/src/components/quickstarts/DockerSpinup.astro:58]: I agree for .env files.

I don't think that's true for kickstart files. Kickstarts are designed to be check into version control.

File: astro/src/components/quickstarts/DockerSpinup.astro [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+<Aside type="note">
+    <p>If you ever want to reset the FusionAuth application, you need to delete the volumes created by Docker Compose by executing <code style="white-space: nowrap">docker compose down -v</code>, then re-run <code>docker compose up -d</code>.</p>
+</Aside>
+
+<p>FusionAuth will be initially configured with these settings:</p>
+
+<ul>
+    <li>Your client Id is <code><RemoteValue url={kickstartUri} selector="$.variables.applicationId" /></code>.</li>
+    <li>Your client secret is <code><RemoteValue url={kickstartUri} selector={($) => $.variables.clientSecret || $.requests.find(r => r.url === '/api/application/#{applicationId}').body.application.oauthConfiguration.clientSecret} /></code>.</li>
```
Comment (mooreds) [inline astro/src/components/quickstarts/DockerSpinup.astro:37]: sick

File: astro/src/components/quickstarts/DockerSpinup.astro [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    {(withTeller)
```
Comment (mooreds) [inline astro/src/components/quickstarts/DockerSpinup.astro:38]: Can we refactor this to be `api` or not `api`? That is the only quickstart that will have a teller/customer and would be more maintainable.

File: astro/src/components/quickstarts/NextSteps.astro [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<p>This quickstart is a great way to get a proof of concept up and running quickly, but to run your application in production, there are some things you're going to want to do.</p>
+
+<h3>FusionAuth Customization</h3>
```
Comment (mooreds) [inline astro/src/components/quickstarts/NextSteps.astro:3]: Bummer these are missing from the TOC.

File: astro/src/components/quickstarts/DockerSpinup.astro (hunk 2 above)
Comment (vcampitelli) [reply astro/src/components/quickstarts/DockerSpinup.astro:58]: Yup, I agree. I got this from some existing quickstarts (e.g. Django, Flask, Remix, PHP), but I'll reword it to only mention `.env`.

**Comment by vcampitelli** [comment]
@mooreds I've applied your suggestions and updated the quickstart instructions at https://github.com/FusionAuth/fusionauth-example-template/pull/36

**Review by mooreds — APPROVED** [review]
LGTM
